examples/cellar/gen/http/swagger/server/server.go

```diff
@@ -9,11 +9,46 @@
 package server
 
 import (
+	"context"
 	"net/http"
 
+	swagger "goa.design/goa/examples/cellar/gen/swagger"
 	goahttp "goa.design/goa/http"
 )
 
+// Server lists the swagger service endpoint HTTP handlers.
+type Server struct {
+	Mounts []*MountPoint
+}
+
+// MountPoint holds information about the mounted endpoints.
+type MountPoint struct {
+	// Method is the name of the service method served by the mounted HTTP handler.
+	Method string
+	// Verb is the HTTP method used to match requests to the mounted handler.
+	Verb string
+	// Pattern is the HTTP request path pattern used to match requests to the
+	// mounted handler.
+	Pattern string
+}
+
+// New instantiates HTTP handlers for all the swagger service endpoints.
+func New(
+	e *swagger.Endpoints,
+	mux goahttp.Muxer,
+	dec func(*http.Request) goahttp.Decoder,
+	enc func(context.Context, http.ResponseWriter) goahttp.Encoder,
+) *Server {
+	return &Server{
+		Mounts: []*MountPoint{
+			{"../../gen/http/openapi.json", "GET", "/swagger/swagger.json"},
+		},
+	}
+}
+
+// Service returns the name of the service served.
+func (s *Server) Service() string { return "swagger" }
```

Comment:
Do we need this function?

Reply:
Yes.. It is used here - https://github.com/goadesign/goa/blob/bc1a832c804fb1ebca28ab8f4289b213c1ce3f00/examples/cellar/cmd/cellarsvc/main.go#L154